src/borrowing/interior-mutability.md

```diff
@@ -2,73 +2,59 @@
 minutes: 10
 ---
 
-<!-- NOTES:
-Introduce the concept, with an example based on Mutex showing an `&self` method doing mutation; reference Cell/RefCell without detail.
--->
-
 # Interior Mutability
 
-Rust provides a few safe means of modifying a value given only a shared
-reference to that value. All of these replace compile-time checks with runtime
-checks.
-
-## `Cell` and `RefCell`
+In some situations, exclusive (`mut`) access cannot be represented at compile
+time. For example, data might be accessed from multiple threads. The "interior
+mutability" pattern allows exclusive access behind a shared reference, using a
+runtime check to enforce the borrowing rules.
```

Comment:
For something like `Cell`, this is not actually 100% true. I believe `Cell` has no runtime overhead: because it does not let you get a reference to the contained value, it doesn't have to do any checks when you `get` or `set` it. The restriction is that it's less useful for non-`Copy` types.

Reply:
This is why I didn't want to use Cell to introduce interior mutability..

Reply:
Right, it's quite subtle and somewhat besides the point! Perhaps you could just drop talking about runtime cost here?